feeds/wlan-ap/opensync/src/platform/openwrt/rootfs/common/etc/init.d/opensync

```diff
@@ -40,8 +40,14 @@ start_service() {
     echo "Checking Redirector"
     redirector=$(uci get system.tip.redirector)
     if [ -z "$redirector" ]; then
-        logger -t opensync "Contacting DigiCert for redirector address"
-        wlan_ap_redirector.sh
+        redirector=$(<"/usr/opensync/certs")
```

Comment:
This should be /usr/opensync/certs/redirector.txt instead of /usr/opensync/certs

Reply:
Good catch.  Thank you.  It was going back to DigiCert to get the redirector.